src/tr1d1um/http.go

```diff
@@ -2,23 +2,262 @@ package main
 
 import (
 	"net/http"
+	ts "github.comcast.com/webpa/tscommon"
+	"github.com/gorilla/mux"
+	"strings"
+	"encoding/json"
+	"regexp"
 	"time"
-	"io/ioutil"
+	"bytes"
+	"fmt"
 )
+var paramRegex *regexp.Regexp
 
+/*
 type ConversionHandler struct {
-	//todo add loggers
+	//todo add //loggers
 	timeOut time.Duration
 	targetUlr string
 }
 
-func (sh *ConversionHandler) ServeHTTP(response http.ResponseWriter, request *http.Request) {
+func (sh ConversionHandler) ServeHTTP(response http.ResponseWriter, request *http.Request) {
 	_ , err := HttpRequestToWRP(request, ioutil.ReadAll)
```

Comment:
We shouldn't convert directly to WRP, but instead convert from the WDMP format into a WRP.